internal/controller/instaslice_daemonset.go

```diff
@@ -762,6 +770,18 @@ func (r *InstaSliceDaemonsetReconciler) discoverMigEnabledGpuWithSlices() ([]str
 	return discoveredGpusOnHost, nil
 }
 
+func (r *InstaSliceDaemonsetReconciler) classicalResourcesOnNode(ctx context.Context, nodeName string) (int64, int64, error) {
+	node := &v1.Node{}
+	if err := r.Client.Get(ctx, client.ObjectKey{Name: nodeName}, node); err != nil {
+		log.FromContext(ctx).Error(err, "unable to retrieve cpu and memory resource on the node")
+	}
+	cpu := node.Status.Capacity[v1.ResourceCPU]
```

Comment:
The actual amount of resources available for the workload on the node is [node.Status.Allocatable](https://github.com/kubernetes/kubernetes/blob/ab26ad095029eb46ca2e938ac3a16cb6f164e657/pkg/apis/core/types.go#L5097) 

`Allocatable = Capacity - System Reserved - Kube Reserved - eviction hard`

Reply:
Thanks, I will use Allocatable